AdvancedTeleport-Bukkit/src/main/java/io/github/niestrat99/advancedteleport/api/AdvancedTeleportAPI.java

```diff
@@ -154,7 +154,7 @@ public static boolean isWarpSet(@NotNull String name) {
             return SpawnSQLManager.get().addSpawn(spawn).thenApplyAsync(x -> {
 
                 // Call the post-create event to indicate success
-                new SpawnPostCreateEvent(spawn).callEvent();
+                Bukkit.getPluginManager().callEvent(new SpawnPostCreateEvent(spawn));
```

Comment:
This could be reverted as the function was added to the CancellableATEvent class.

Reply:
Only issue with that is SpawnPostCreateEvent is not cancellable lmao, this event is used to pass the created Spawn object after it's been created and added to the database.